tests/metagpt/utils/test_s3.py

```diff
@@ -41,17 +41,15 @@ async def test_s3():
     res = await conn.cache(data, ".bak", "script")
     assert "http" in res
 
-
-@pytest.mark.asyncio
-async def test_s3_no_error():
+    key = CONFIG.S3_ACCESS_KEY
+    CONFIG.S3_ACCESS_KEY = "YOUR_S3_ACCESS_KEY"
```

Comment:
In a general sense, CONFIG is a read-only class and should not be writable

Reply:
1. `CONFIG` is a session-scoped context that supports both read and write operations. 
2. `Config` represents a configuration file and is read-only. 
3. The root cause of the issue here is the absence of the concept of a session in the console version. Therefore, the name `CONFIG` is used. 
4. To support the concept of a session functionally, the `contextvars.ContextVar` data type has been employed.

Reply:
There's going to be some tricky tricky about that. The associated code needs to be refactored to be more straightforward. If it is a session concept, you should provide a session object, and use session.config or context.config in the session object to accurately express the context in which it is located. It needs to be refactored in the future, so you can put it here first